src/batched/KokkosBatched_QR_WithColumnPivoting_TeamVector_Internal.hpp

```diff
@@ -171,6 +172,7 @@ namespace KokkosBatched {
 						    n_A22,
 						    A_part3x3.A12, as1,
 						    norm_part1x3.A2, 1);
+    member.team_barrier();
```

Comment:
norm_part1x3 values used to update norm_part1x2 with "norm_part1x2.mergeToAL(norm_part1x3);" 
TeamVectorUpdateColumnNormsInternal has no reduction, just a parallel_for TeamVector, so this could potentially not be complete when the next calls are made.